kanidmd_web_ui/src/views/security.rs

```diff
@@ -111,6 +121,36 @@ impl Component for SecurityApp {
             }
             Msg::Error { emsg, kopid } => {
                 self.state = State::Error { emsg, kopid };
+                self.unix_input_value = "".to_string();
+                true
+            }
+            Msg::Success(msg) => {
+                self.state = State::Success { msg };
+                self.unix_input_value = "".to_string();
+                true
+            }
+            Msg::ChangeUnixPassword => {
+                let token = ctx.props().token.clone();
+
+                let jwtu =
+                    JwsUnverified::from_str(&token).expect_throw("Invalid UAT, unable to parse");
+
+                let uat: Jws<UserAuthToken> = jwtu
+                    .unsafe_release_without_verification()
+                    .expect_throw("Unvalid UAT, unable to release ");
+
+                let id = uat.inner.uuid.to_string();
+                let newpw = self.unix_input_value.clone();
+                ctx.link().send_future(async {
+                    match Self::update_unix_password(id, token, newpw).await {
+                        Ok(v) => v,
+                        Err(v) => v.into(),
+                    }
+                });
+                true
+            }
+            Msg::UnixPasswordInput(mut inputvalue) => {
+                std::mem::swap(&mut self.unix_input_value, &mut inputvalue);
```

Comment:
Why not just use an assignment here? That feels weird.

Reply:
I've taken this from the input field on the login page. I'm pretty new to this framework so I assumed it has a reason. Will have to change this anyways when switching to options.

Reply:
Also obsolete now